day_7/high_level.md

```diff
@@ -0,0 +1,21 @@
+High Level Write Up
+
+I chose the Checkerboard project to attempt an explanation.
+
+puts a question to the user asking them how large they would like their checker_board
+
+assigns a name for user input and gets input from user in the form of an integer
+
+assigns and array to the board as "X" for a black square and " " for a white square
+
+  tells program the number of times to repeat integer given by user to build elements of the checker_board
+
+    changes the board array from an array to a hash to call array's elements individually as a key and hash
+      puts the key and value in order and repeats using the integer value
+      puts the key and value in reverse order and repeats using the integer value
+    end
+end
```

Comment:
This is a good start to breaking down your algorithm. I think if you went more detailed and re-referenced the end goal of what you're looking to print out, you'd get there!